### PR TITLE
Pickup, put down animations.

### PR DESCRIPTION
**Pull request by LudwigVonChesterfield**
Don't credit me, the idea is ported from Oracle station.

More animations! Now putting stuff on tables, closets, racks, disposal bins, is... Animated! Hooray!

### CODE REVIEW

File: code/modules/tables/interactions.dm [hunk 1]
```diff
@@ -135,7 +135,7 @@
 		user << SPAN_WARNING("There's nothing to put \the [W] on! Try adding plating to \the [src] first.")
 		return
 
-	if (user.unEquip(W, src.loc))
-		set_pixel_click_offset(W, params)
+	set_pixel_click_offset(W, params)
+	user.unEquip(W, src.loc)
```
Comment (NanakoAC) [inline code/modules/tables/interactions.dm]: nooop

this is a safety check. unEquip returns true if it succeeds, and THEN we move the object
these two should be restored to how they were

Comment (LudwigVonChesterfield) [reply code/modules/tables/interactions.dm]: There are no cases in this very situation, when it will return FALSE. Prove me wrong.

Comment (LudwigVonChesterfield) [reply code/modules/tables/interactions.dm]: (P.S.: And even if it does, the item will stay in hand, and there it doesn't matter what it's pixel_x and pixel_y is).

**Comment by LudwigVonChesterfield** [comment]
I however see it as "beauty is in the eye of the beholder". And haven't been given any reason to not switch around the safety check.

**Pull request merged**